### PR TITLE
[11.0][FIX] business_requirement: Security

### PR DESCRIPTION
**Pull request by Tardo**
Superseeds: #322 

Now users without permissions over BR can access to '/my' page.
"Business Requirements" only can be seen by BR users & Portal users in '/my' page

Cc @tecnativa

### CODE REVIEW

File: business_requirement/controllers/portal.py [hunk 1]
```diff
@@ -12,11 +12,13 @@ class CustomerPortal(CustomerPortal):
 
     def _prepare_portal_layout_values(self):
         values = super(CustomerPortal, self)._prepare_portal_layout_values()
-        br_count = request.env['business.requirement'].search_count(
-            self._prepare_br_base_domain())
-        values.update({
-            'business_requirement_count': br_count,
-        })
+        BRObj = request.env['business.requirement']
```
Comment (martintorresvm) [inline business_requirement/controllers/portal.py]: snake case for variables 
```suggestion
        br_obj = request.env['business.requirement']
```

**Review by martintorresvm — CHANGES_REQUESTED** [review]
Need to increase number of version.
Others LGMT also tested on runbot

Fix #321 issue

**Comment by pedrobaeza** [comment]
@victormmtorres version number is increased by bot. No need of doing it manually.

File: business_requirement/controllers/portal.py (hunk 1 above)
Comment (pedrobaeza) [reply business_requirement/controllers/portal.py]: User either `BR` or `br_obj`, but not a mix

**Comment by pedrobaeza** [comment]
@Tardo always remember to comment when you perform the changes.

File: business_requirement/controllers/portal.py (hunk 1 above)
Comment (Tardo) [reply business_requirement/controllers/portal.py]: @victormmtorres I all times use UpperCamelCase for this kind of vars... You can see it in other parts of Odoo code.

Comment (pedrobaeza) [reply business_requirement/controllers/portal.py]: @Tardo, `BrObj` would be UpperCamelCase, but not `BRObj`.

**Comment by Tardo** [comment]
@victormmtorres @pedrobaeza changes done. To follow the style of @victormmtorres (that is the main developer of this migration) now uses snake case.

**Review by martintorresvm — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
/ocabot merge minor

Please forward-port it to v12.

**Pull request closed without merging**